### DISCUSSION TITLE
🚀 GitHub Student Pack Approved but Benefits Not Received After 2 Days – Need Assistance!

### DISCUSSION THREAD

**Discussion by Ozomozyan** [community/community#144649] [GitHub Education] [answered]
### Hi everyone,

Hello GitHub Community,

My GitHub Student Pack application was recently approved!  However, it’s been two days since the approval, and I haven’t received access to the benefits yet.

Here’s a brief overview:

Application Status: Approved
Approval Date: 13/11/2024
Steps Taken So Far:
Checked my email (including spam/junk folders)
Reviewed GitHub’s [Student Pack FAQs](https://education.github.com/pack)

What I’m Seeking:

Guidance on potential reasons for the delay
Steps I can take to expedite the process
Any similar experiences from the community and how they were resolved
I’m eager to start leveraging the GitHub Student Pack to enhance my projects and learning journey. Any assistance or advice would be greatly appreciated!

Thank you in advance! 🙏

**Comment by davevad93** [comment, marked as answer]
Hi @Ozomozyan ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ozomozyan was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
